packages/main/src/plugin/api/cli-tool-info.ts

```diff
@@ -31,4 +31,6 @@ export interface CliToolInfo {
   state: CliToolState;
   images?: ProviderImages;
   extensionInfo: CliToolExtensionInfo;
+  version: string;
+  path: string;
```

Comment:
```suggestion
  path?: string;
```

Reply:
Nit: Registration of new tool not necessarily have location and version, because it could be unknown at the time of registration. That would save couple lines when registering a tool, because there would be no path and version fields initialized with empty string.

Reply:
![image](https://github.com/containers/podman-desktop/assets/620330/f2dce6e2-7975-499c-b874-04cb58cb8967)

There also should be `* SETUP NEEDED` or `* ACTIVE` on the left side of divider to show if CLI tool is detected or need installation.